rero_ils/accounts_views.py

```diff
@@ -71,23 +104,51 @@ def post(self, **kwargs):
         self.login_user(user)
         return self.success_response(user)
 
+    def verify_login(self, user, password=None, **kwargs):
+        """Verify the login via password."""
+        if not user.password or not verify_and_update_password(password, user):
+            _abort(_('INVALID_USER_OR_PASSWORD'))
```

Comment:
You mark the string `INVALID_USER_OR_PASSWORD` for translation ? Why use a snake case string ?
maybe it should be `get_message('INVALID_USER_OR_PASSWORD')[0]` ?